modules/phase_field/src/ics/MultiSmoothSuperellipsoidIC.C

```diff
@@ -229,7 +248,7 @@ MultiSmoothSuperellipsoidIC::overlapCheck(const Point & newcenter, Real nc_as, R
   }
   //Check for overlap between extremes of new ellipsoid candidate and the center
```

Comment:
Add empty line above here and please fix the broken indentation (not introduced by this PR).